drivers/led/lp5562.c

```diff
@@ -925,6 +994,31 @@ static int lp5562_led_init(const struct device *dev)
 		return -EIO;
 	}
 
+	result = lp5562_verify_config(config);
+	if (result != 0) {
+		return result;
+	}
+
+	result = lp5562_led_update_current(dev, LP5562_CHANNEL_R);
+	if (result != 0) {
+		return result;
+	}
+
+	result = lp5562_led_update_current(dev, LP5562_CHANNEL_G);
+	if (result != 0) {
+		return result;
+	}
+
+	result = lp5562_led_update_current(dev, LP5562_CHANNEL_B);
+	if (result != 0) {
+		return result;
+	}
+
+	result = lp5562_led_update_current(dev, LP5562_CHANNEL_W);
+	if (result != 0) {
+		return result;
+	}
```

Comment:
Since all the current I2C registers are adjacent, and the driver doesn't support updating the current value at runtime, then maybe we should write them all at once with a single `i2c_write_dt` call ?

Reply:
not all of them (white is a little bit further away) but 2 transactions is better then 4